### PR TITLE
Update newLint.sh script and contributing guide. 

### PR DESCRIPTION
**Pull request by pkoro**
#383

### CODE REVIEW

File: newLint.sh [hunk 1]
```diff
@@ -17,18 +18,32 @@ if [ $# -eq 1 ]; then
...
+if [ $# -eq 2 ]; then
+    echo "Not enough arguments provided..."
+    echo "$USAGE"
+    exit 1
+fi
+
+if [ ! -d lints/$1 ]
 then
-   echo "File already exists. Can't make new file."
+   echo "Directory $1 does not exist. Can't make new file."
```
Comment (cpu) [inline newLint.sh]:
```suggestion
   echo "Directory 'lints/$1' does not exist. Can't make new file."
```

File: template [hunk 2]
```diff
@@ -12,7 +12,7 @@
  * permissions and limitations under the License.
  */
 
-package lints
+package PACKAGE
 
 import (
 	"github.com/zmap/zcrypto/x509"
```
Comment (cpu) [inline template]: Would you mind also adding the missing `github.com/zmap/zlint/lint` import to the template?

File: newLint.sh [hunk 3]
```diff
@@ -17,18 +18,32 @@ if [ $# -eq 1 ]; then
...
-cp template lints/lint_$FILENAME.go
+if [ -e lints/$1/lint_$2.go ]
+then
+   echo "File already exists. Can't make new file."
+   exit 1
+fi
+
+PATHNAME=$1
+LINTNAME=$2
+FILENAME=${LINTNAME:2}
```
Comment (cpu) [inline newLint.sh]: Am I correct in understanding the `LINTNAME`/`FILENAME` split here is being used to remove the `e_`, `w_` etc prefix from lint filenames? If so can you add a comment to that effect? I think it's a little bit subtle as is.

File: newLint.sh (hunk 1 above)
Comment (pkoro) [reply newLint.sh]: Done!

File: newLint.sh (hunk 3 above)
Comment (pkoro) [reply newLint.sh]: Yes you are right. I added one comment regarding this.

**Pull request merged**